packages/gatsby/src/internal-plugins/prod-404/gatsby-node.js

```diff
@@ -4,18 +4,19 @@ const { boundActionCreators } = require(`../../redux/actions`)
 const PROD_404_PAGE_PATH = `/404.html`
 
 let page404 = null
-exports.onCreatePage = ({ page, store, actions }) => {
+const fourOhFourRegex = /^\/?404\/?$/
```

Comment:
delicious variable naming :D

Reply:
:-D one of my small joys is making up funny variable names